test/http/doc_examples.cpp

```diff
@@ -307,6 +311,50 @@ class doc_http_samples_test
         BEAST_EXPECTS(! ec, ec.message());
     }
 
+    void
+    doConstAndMutableBody()
+    {
+        test::pipe c{ios_};
```

Comment:
Wow yeah, you understand that `test::pipe` mumbo-jumbo and you even wrote the tests - this is great!